guidelines/coding-guidelines.md

```diff
@@ -11,3 +11,59 @@ For public facing interfaces we favor read-only collections and enumerables. For
 * Avoid using `System.Linq`
 * Avoid using `foreach` over collections that do not have a struct enumerator
 
+## Dependencies
+
+### 3rd party dependencies
+
+#### Package dependencies
+
+We want to compile against the lowest major to catch breaking changes
+
+PackageReference: `[Major.0.0, Major+1)`
```

Comment:
Is there ever any value in upping the minimum patch version to avoid a bug in the third party package? Would it help in guiding our customers further toward the pit of success?